templates/libreswan.connection.conf

```diff
@@ -0,0 +1,24 @@
+#jinja2: lstrip_blocks: True
+{% for connection in vpn_connections %}
+{%   if item in connection.hosts %}
+conn {{ inventory_hostname + '-to-' +  item }}
+  left={{ ansible_host }}
+  right={{ hostvars[item].ansible_host }}
+{%     if 'auto' in connection %}
+  auto={{ connection.auto }}
+{%     endif %}
+{%     if 'mode' in connection %}
+  type={{ connection.mode }}
+{%     endif %}
+{%     if 'rekey' in connection %}
+  rekey={% if connection.rekey %}yes{% else %}no{% endif %}
```

Comment:
use the `ternary` filter here - https://docs.ansible.com/ansible/2.9/user_guide/playbooks_filters.html#id8
```
   rekey={{ connection.rekey | ternary('yes', 'no') }}
```
also below where you use a similar construct